code/modules/surgery/organs/liver.dm

```diff
@@ -86,3 +86,21 @@
 			damage+=100
 		if(2)
 			damage+=50
+
+/obj/item/organ/liver/ipc  // It's a liver because a liver is needed to process chems (Like holywater) but it doesn't make sense for a robot to have a liver.

```

Comment:
Not a fan of this explanation. When you take out an IPC's "Morality Core" they suddenly can't process chems?

I'd call it a Substance Processor or something, so that it makes sense.